flake.nix

```diff
@@ -0,0 +1,39 @@
+{
+  inputs = {
+    nixpkgs.follows = "tullia/nixpkgs";
+    std.follows = "tullia/std";
+    tullia.url = github:input-output-hk/tullia;
+  };
+
+  outputs = {
+    self,
+    std,
+    tullia,
+    ...
+  } @ inputs:
+    std.growOn {
+      inherit inputs;
+      cellsFrom = ./nix;
+      cellBlocks = [
+        (std.functions "library")
+        (std.functions "hydraJobs")
+        (tullia.tasks "pipelines")
+        (std.functions "actions")
+      ];
+    }
+    (
+      tullia.fromStd {
+        actions = std.harvest self ["cloud" "actions"];
+        tasks = std.harvest self ["automation" "pipelines"];
+      }
+    )
+    {
+      hydraJobs = std.harvest self ["automation" "hydraJobs"];
+    };
+
+  nixConfig = {
+    extra-substituters = ["https://cache.iog.io"];
```

Comment:
in the current [ci workflow](https://github.com/input-output-hk/hydra/blob/master/.github/workflows/ci.yaml#L37) we define more than 1 substituter.

does it make sense to you to define them here as well?